autoprotocol/builders.py

```diff
@@ -1892,22 +1977,22 @@ def get_mode_params(self, mode, mode_params):
             "spin_acceleration": "g"
         }
         mode_param_output = {}
-        speed_param = [(k) for k in mode_params.keys()
-                       if k in speed_unit_dict.keys()]
+        speed_param = [key for key in mode_params.keys()
```

Comment:
open a separate PR to commit refactoring

Reply:
ya... this was another lint error thing. reverting.